env/requirements-test.txt

```diff
@@ -3,3 +3,4 @@ pytest
 pytest-cov
 coverage
 pymap3d>=2.9.0
+scipy
```

Comment:
With this, Scipy is now a runtime requirement so it should be added to `pyproject.toml` instead of here. When you do, please set the lower version boundary to `>=1.0.8` since that's the minimum we need, right?

Reply:
I added scipy to pyroject.toml and removed it from requirements-test.txt

Reply:
and the minimum version is 1.8.0, not 1.0.8 as I stated earlier...

Reply:
👍🏾